### ISSUE TITLE
Failed to upgrade system from lineage-downloads

### ISSUE THREAD

**Issue by dmknght** [arian-ota/ota#26]
I noticed update image switched from build on this github to lineage-downloads. I tried update from it, however I faced some issues:
1. Download progress interrupts every time file downloaded roughly 1gb. I managed to be able to download the file after 2 days tried
2. My phone refused to install this new update (zip file). I'm sure I downloaded correct version (mondrian for my K60). I tried both push update and local update (copy file to my phone).

**Comment by dmknght** [comment]
That was my bad. Apparently the solution to update mentioned in here as "migration"
https://xdaforums.com/t/rom-14-unofficial-lineageos-21-for-poco-f5-pro-redmi-k60.4641835/page-6

Seems like we have to run adb commands, then push update via recovery

**Comment by dmknght** [comment]
By now, there are 3 different download locations:
1. This repo (April and earlier)
2. https://github.com/cupid-development/ota/releases (builds in May)
3. https://lineage-downloads.mainlining.org/devices/mondrian/builds latest builds of May and June

As I tried, latest build didn't work (signature error). I had to use the method in the link above, used latest build in cupid development to upgrade, then download latest version in 3rd link to do local upgrade
